service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -89,6 +89,10 @@ protected boolean doRegister() {
 
     private void checkSchemaIdSet() {
         Microservice existMicroservice = srClient.getMicroservice(microservice.getServiceId());
+        if (existMicroservice == null) {
```

Comment:
i feel that, it should be treat as register microservice failed, and reset the service center task like register instance failed.